src/header/Header.js

```diff
@@ -32,10 +27,12 @@ function Header() {
           className="burger-menu__icon"
           src={menu}
           alt="Burger menu icon"
-          onClick={menuHandler}
+          onClick={() => {
+            setOpen(!open);
+          }}
         />
 
-        <ul className="burger-menu__list list hide">
+        <ul className={`burger-menu__list list ${open ? `open` : `close`}`}>
```

Comment:
Minor thing, you can use single quotes inside ternaries.

`${open ? 'open' : 'close'}`